change/@fluentui-react-tag-picker-4e44aceb-6929-46a4-9e5a-299fa05bcfce.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32188&commitId=94d5aecd0a3a697fde0b30c2104b1b83cbcd8e62&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>